pom.xml

```diff
@@ -140,6 +140,12 @@
             <version>${junit-jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Look like 
```
jnit-jupiter-engine
```
already include
```
jnit-jupiter-api
```
and the dependency of junit-jupiter-api and junit-jupiter-engine can change to 
```
<dependency>
    <groupId>org.junit.jupiter</groupId>
    <artifactId>junit-jupiter</artifactId>
    <version>${junit-jupiter.version}</version>
    <scope>test</scope>
</dependency>
```
The dependency diagram
![image](https://user-images.githubusercontent.com/4516752/132282575-c7ef5085-da55-455b-afef-5ea6164103d0.png)

Reply:
fixed